test/eth_getUncles.js

```diff
@@ -0,0 +1,113 @@
+      let testBlock = await forkedWeb3.eth.getBlock(oneUncleBlockNumber);
+      assert.strictEqual(testBlock.uncles[0], validSingleUncleId.position0);
+
+      // Validate single uncle id using Ganache rpc methods
+      let method = "eth_getUncleByBlockNumberAndIndex";
+      let params = [oneUncleBlockNumber, 0];
+      let { result } = await rpcSend(method, params, forkedWeb3);
+      assert.strictEqual(result, validSingleUncleId.position0);
+
+      method = "eth_getUncleByBlockHashAndIndex";
+      params = [testBlock.hash, 0];
+      ({ result } = await rpcSend(method, params, forkedWeb3));
+      assert.strictEqual(result, validSingleUncleId.position0);
+
+      // Access index greater than position 1 by block hash
+      try {
+        method = "eth_getUncleByBlockHashAndIndex";
+        params = [testBlock.hash, 2];
+        ({ result } = await rpcSend(method, params, forkedWeb3));
+        assert.fail("Should not be able to access an uncle index greater than 1");
+      } catch (error) {
+        assert.strictEqual(error.message, "Uncle array index must be 0 or 1");
+      }
+
+      // Access index greater than position 1 by block number
+      try {
+        method = "eth_getUncleByBlockNumberAndIndex";
+        params = [testBlock.hash, 2];
+        ({ result } = await rpcSend(method, params, forkedWeb3));
+        assert.fail("Should not be able to access an uncle index greater than 1");
+      } catch (error) {
+        assert.strictEqual(error.message, "Uncle array index must be 0 or 1");
+      }
+
+      // Attempt to access an out of range index
+      try {
+        method = "eth_getUncleByBlockHashAndIndex";
+        params = [testBlock.hash, 1];
+        ({ result } = await rpcSend(method, params, forkedWeb3));
+        assert.fail("Should not be able to access an out of range index");
+      } catch (error) {
+        assert.strictEqual(error.message, "Uncle array index is out of range");
+      }
+    });
+
+    it("should validate a block with a DOUBLE uncle id by hash or block number reference", async function() {
+      this.timeout(10000);
+      // Directly validate a double uncle count from a know block number on Mainnet
+      const testBlock = await forkedWeb3.eth.getBlock(twoUncleBlockNumber);
+      assert.strictEqual(testBlock.uncles.length, 2);
+
+      // Validate single uncle id using Ganache rpc methods
+      let method = "eth_getUncleByBlockNumberAndIndex";
+      let params = [twoUncleBlockNumber, 0];
+      let { result } = await rpcSend(method, params, forkedWeb3);
+      assert.strictEqual(result, validDoubleUncleIds.position0);
+
+      method = "eth_getUncleByBlockHashAndIndex";
+      params = [testBlock.hash, 0];
+      ({ result } = await rpcSend(method, params, forkedWeb3));
+      assert.strictEqual(result, validDoubleUncleIds.position0);
+
+      // Validate single uncle id using Ganache rpc methods
+      method = "eth_getUncleByBlockNumberAndIndex";
+      params = [twoUncleBlockNumber, 1];
```

Comment:
If you use `"0x1"` here instead of 1 does everything still work? Can you add a test proving this? Same with the block number -- make sure it works with hex encoding and provide a test.